iroha_python/iroha2/__init__.py

```diff
@@ -1,15 +1,33 @@
 #!/usr/bin/env python
 from .sys.rust import *
 from .sys import KeyPair, Client as _Client
+from .data_model.query import Query as _Query
 
 
 class Client:
     def __init__(self, cfg):
         self.cl = _Client(cfg)
 
+    @property
+    def account(self):
+        return self.cl.account
+
     def submit_tx(self, tx: list):
         tx = [i.to_rust() for i in tx]
         return self.cl.submit_all_with_metadata(tx, {})
 
     def submit_inst(self, inst):
-        return self.cl.submit_all_with_metadata([inst.to_rust()], {})
+        return self.submit_tx([inst])
+
+    def submit_tx_blocking(self, tx: list):
+        tx = [i.to_rust() for i in tx]
+        return self.cl.submit_all_blocking_with_metadata(tx, {})
+
+    def submit_inst_blocking(self, inst):
```

Comment:
Let's use full name for `instruction`, sorry for not telling it earlier

Reply:
For function names, right? May be we can rename it to `isi`?